### ISSUE TITLE
Unnecessary aborts from the MDP executor?

### ISSUE THREAD

**Issue by hawesie** [strands-project/strands_executive#57]
@ToMadoRe reported a few of these:

```
[ERROR] [WallTime: 1399960961.602131] State transition is not in MDP model! Aborting...
```

They can occur when the human presses a button on the screen to send the robot to a particular point.  

Can these happen when it really is possible to navigate between the two points? If so, I think it might be more robust to have a fallback position rather than aborting. This would make the robot more usable at AAF currently.

**Comment by bfalacerda** [comment]
There is a bug there as it is outputting this too often in the early stages of deployment, I'll look into in tomorrow. However if the send robot somewhere button is preemptive the actions being executed and issuing a new navigation goal, it won't happen there. @ToMadoRe is this still happening at this point?

**Comment by bfalacerda** [comment]
solved in #59
